app/views/users/_loggedin.html.erb

```diff
@@ -1,17 +1,31 @@
-<div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url('<%= @user.profile_image_path %>')">
-  <div class="container">
-    <h1><%= @user.full_name %>'s profile</h1>
-<!-- if user is available show "search button" -->
-    <% if @user.available %>
-      <% if @user.user_adjectives.present? %>
-              <p>Looking for</p><p><%= @user.user_adjectives.first[:adjective] %>, <%= @user.user_adjectives.first[:adjective_2] %> and <%= @user.user_adjectives.first[:adjective_3] %></p>
-        <%= link_to "edit my search", edit_user_user_adjective_path(@user, @user.user_adjective_ids.first), class:"main_action_button mt-3"  %>
-      <% else %>
-        <%= link_to "my search", new_user_user_adjective_path(@user, @user.user_adjective_ids.first), class:"main_action_button mt-3"  %>
-      <% end %>
-    <% end %>
+<div class="container">
+  <div class="row">
+      <div class="col-sm-1">
+      </div>
+      <div class="col-sm-4">
+        <%= cl_image_tag(@user.profile_image_path, gravity:"faces", crop:"fill", class:"profile-picture") %>
+      </div>
+      <div class="col-sm-6 catch-paragraph-left">
+        <%= @user.first_name %>
+        <!-- if user is available show "search button" -->
+          <% if @user.available %>
+            <% if @user.user_adjectives.present? %>
+              looking for someone who is <%= @user.user_adjectives.first[:adjective] %>, <%= @user.user_adjectives.first[:adjective_2] %> and <%= @user.user_adjectives.first[:adjective_3] %>
+              <% else %>
+              <%= link_to "my search", new_user_user_adjective_path(@user, @user.user_adjective_ids.first), class:"main_action_button mt-3"  %>
```

Comment:
should be edit_user_adjective, but i changed the code for that, so would have to check